htdocs/lib/bench.inc.php

```diff
@@ -10,7 +10,9 @@
 //
 class Cbench
 {
+
     public $start;
+
     public $stop;
 
```

Comment:
@teiling88 do you agree with inserting blank lines before all class member variables? So far there are no docblocks in lib1 and lib2.

The blank lines are also created after the opening { of the class declarations:

```php
 class login
 {

     public $userid = 0;

     public $username = '';

     public $permanent = false;
```

I think it's best practice to have no blank lines after { , code sniffer removes them in control blocks.